### PR TITLE
Fix: NoMethodError - undefined method 'tagged' for #<ActiveSupport::Logger>

### PR DESCRIPTION
**Pull request by curvo**
This fixes a bug would be found in versions of Rails applications (<v5) that do not have the default logger set up for tagging.  The suggested fix is to check for the method before re-assigning the default webpacker logger to be the rails.logger.

### CODE REVIEW

**Comment by curvo** [comment]
screenshot 2017-08-30 18 54 31

File: lib/webpacker/railtie.rb [hunk 1]
```diff
@@ -24,7 +24,9 @@ class Webpacker::Engine < ::Rails::Engine
 
   initializer "webpacker.logger" do
     config.after_initialize do |app|
-      Webpacker.logger = ::Rails.logger
+      if ::Rails.logger.respond_to?(:tagged)
+        Webpacker.logger = ::Rails.logger
+      end
```
Comment (javan) [inline lib/webpacker/railtie.rb]: Could create a tagged logger here to preserve the original logger:
```ruby
if ::Rails.logger.respond_to?(:tagged)
  Webpacker.logger = ::Rails.logger
else
  Webpacker.logger = ActiveSupport::TaggedLogging.new(::Rails.logger)
end
```

Comment (gauravtiwari) [reply lib/webpacker/railtie.rb]: I guess we are already doing this here right? https://github.com/rails/webpacker/blob/68b9f086c1d262be895c2750cb3a25879157733a/lib/webpacker/instance.rb#L2

Comment (javan) [reply lib/webpacker/railtie.rb]: That creates a brand new logger. Doesn't wrap your existing `Rails.logger` instance.

Comment (gauravtiwari) [reply lib/webpacker/railtie.rb]: Ahh I see, totally missed 👍

Comment (curvo) [reply lib/webpacker/railtie.rb]: Makes sense... I'll push that today.

**Comment by gauravtiwari** [comment]
thanks  🍰

**Comment by curvo** [comment]
Happy to contribute!

**Pull request merged**